test/CompanyService.Business.UnitTests/Commands/ContractSubject/CreateContractSubjectCommandTests.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using FluentValidation.Results;
+using LT.DigitalOffice.CompanyService.Business.Commands.ContractSubject;
+using LT.DigitalOffice.CompanyService.Business.Commands.ContractSubject.Interfaces;
+using LT.DigitalOffice.CompanyService.Data.Interfaces;
+using LT.DigitalOffice.CompanyService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.CompanyService.Models.Dto.Requests.ContractSubject;
+using LT.DigitalOffice.CompanyService.Validation.ContractSubject.Interfaces;
+using LT.DigitalOffice.Kernel.BrokerSupport.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.UnitTestKernel;
+using Microsoft.AspNetCore.Http;
+using Moq;
+using Moq.AutoMock;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.CompanyService.Business.UnitTests.Commands.ContractSubject;
+
+public class CreateContractSubjectCommandTests
+{
+  private AutoMocker _mocker;
+  private ICreateContractSubjectCommand _command;
+  private CreateContractSubjectRequest _request;
+  private OperationResultResponse<Guid?> _response;
```

Comment:
i think it`s better to rename this to goodResponse or successResponse

Reply:
Done